msmbuilder/tests/test_featurizer.py

```diff
@@ -85,6 +86,19 @@ def test_von_mises_featurizer():
     assert X_all[0].shape == (n_frames, 20), ("unexpected shape returned: (%s, %s)" %
                                               X_all[0].shape)
 
+    #test to make sure results are being put in the right order
+    feat = VonMisesFeaturizer(["phi","psi"], n_bins=10)
+    _, all_phi = md.compute_phi(trajectories[0])
+    X_all = feat.transform(trajectories)
+    all_res=[]
+    for frame in all_phi:
+        for dihedral_value in frame:
+            all_res.extend(vm.pdf(dihedral_value,loc=feat.loc, kappa=feat.kappa))
+    for k in range(10):
```

Comment:
this checks 10 random features to make sure that they appear in the right columns